src/main/webapp/app/exam/exam-scores/exam-scores.component.ts

```diff
@@ -45,6 +48,11 @@ export class ExamScoresComponent implements OnInit, OnDestroy {
     public filterForSubmittedExams = false;
     public filterForNonEmptySubmissions = false;
 
+    @ViewChild('histogram', { static: false })
+    private canvas: ElementRef<HTMLCanvasElement>;
```

Comment:
Is it really necessary to get a reference to the canvas?
Please have a look how this is solved for the quiz charts.
It is not necessary in this case.

Reply:
Example `quiz-statistic.component.html`

<canvas baseChart [datasets]="datasets" [labels]="labels" [options]="options" [chartType]="chartType" [colors]="colors"> </canvas>

In the corresponding ts file: options, labels, colors and chartType are defined and automatically injected by angular